src/jackknife.jl

```diff
@@ -0,0 +1,45 @@
+"""
+Use bootweights to creaye replicate weights using Rao-Wu bootstrap. The function accepts a `SurveyDesign` and returns a `ReplicateDesign` which has additional columns for replicate weights. 

```

Comment:
JKn algorithm docstring and docstest, similar to bootweights. Cite the formula from the theory if appropriate